client/lib/analytics/ad-tracking/setup.js

```diff
@@ -254,4 +255,10 @@ function setupClarityGlobal() {
 		function () {
 			( window.clarity.q = window.clarity.q || [] ).push( arguments );
 		};
+
+	setTimeout( () => {
+		// Identify the user with a custom-id email
+		// https://learn.microsoft.com/en-us/clarity/setup-and-installation/identify-api
+		window.clarity( 'identify', getCurrentUser()?.email );
+	}, 10 );
```

Comment:
Is the `10` here due to we need to give time to load the `getCurrentUser`?

Reply:
Since this function is called before we set the user at calypso-analytics, with this delay we can reliable get the user.